### ISSUE TITLE
Rework header structure on Emory adjusted pages

### ISSUE THREAD

**Issue by erussey** [emory-libraries/aspace#141]
Fix header structure on homepage and help pages to address missing `<h2>` element to optimize screen reader responsiveness.

Affected pages are: 
https://archives-test.libraries.emory.edu/help
https://archives-test.libraries.emory.edu/help-requesting
https://archives-test.libraries.emory.edu/
https://archives-test.libraries.emory.edu/about

**Comment by CB987** [comment]
- couldn't find an error on /help or  /help-requesting
- home page, converted the < p > subtitle in the hero to an < h2 > and changed the styling for that specific element. 

![Screenshot 2023-05-16 at 11.54.05 AM.png](https://images.zenhubusercontent.com/5d379c9d5221e52414fca85c/8622d759-5abc-4b3f-8ef6-c62047e74b2e)

- on /about, there were h2s but no h1. I converted the top h2, "About ArchivesSpace" to an h1. if this is not acceptable, we just need a page title to be the h1, and/or a different title for the first section. Maybe could make the page title just "About" and then do the 3 section titles ArhcivesSpace, Accessibility, etc.

![Screenshot 2023-05-15 at 8.01.50 PM.png](https://images.zenhubusercontent.com/5d379c9d5221e52414fca85c/1eac0ad0-890b-4e92-8c97-886d2bc65089)

**Comment by CB987** [comment]
@erussey before i PR i wanted to get your feedback. i can demo on my local if the screenshots are not clear/helpful.

**Comment by erussey** [comment]
@CB987 : the above is ok (and a simple solution, so thank you).

**Comment by erussey** [comment]
@CB987 : I don't know how to test this?

**Comment by CB987** [comment]
Screenshot from current test site with corrected headings hierarchy, this is not noted as an issue. ![Screenshot 2023-05-23 at 5.00.26 PM.png](https://images.zenhubusercontent.com/5d379c9d5221e52414fca85c/4acb0dc8-e7eb-47ff-9ca1-cbdd434def5f)
On production you can see the issue still exists. 
![Screenshot 2023-05-23 at 4.58.07 PM.png](https://images.zenhubusercontent.com/5d379c9d5221e52414fca85c/ce01df4e-fae6-42e2-a44a-d3db1e94a51d)

**Comment by erussey** [comment]
Great, thanks @CB987 !

**Comment by CB987** [comment]
and on the home page:
prod, with issue noted:
![Screenshot 2023-05-23 at 5.07.08 PM.png](https://images.zenhubusercontent.com/5d379c9d5221e52414fca85c/21f2ef1f-22fe-4394-bb3a-136a30576c56)
test, no longer reports that issue:
[Uploading... Screenshot 2023-05-23 at 5.07.45 PM.png]
